package.json

```diff
@@ -13,21 +13,25 @@
     "ts:lint": "tslint -c tslint.json '**/*.ts'",
     "ts:build": "rm -rf dist/ && tsc -p tsconfig.json",
     "ts:watch": "yarn ts:build --watch",
+    "db:migrate": "./run-migrations.sh",
```

Comment:
as your sequelize-files require a build, I'd suggest to add a pre-db:migrate steps that runs `yarn build`, or put the `yarn build` into the `./run-migrations.sh` — if the intent of this npm-task/shell-script is to provide a one-stop-no-worries command.

Reply:
Done.